firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsController.java

```diff
@@ -872,4 +881,31 @@ private static boolean firebaseCrashExists() {
   }
 
   // endregion
+
+  // region ApplicationExitInfo
+
+  /** If an ApplicationExitInfo exists relevant to the session, writes that event. */
+  private void writeApplicationExitInfoEventIfRelevant(String sessionId) {
+    if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+      ActivityManager activityManager =
+          (ActivityManager) context.getSystemService(Context.ACTIVITY_SERVICE);
+      List<ApplicationExitInfo> applicationExitInfoList =
+          activityManager.getHistoricalProcessExitReasons(null, 0, 0);
```

Comment:
Change maxEvents to 1, and add a TODO to change this later if start linking ApplicationExitInfo to other events.

Reply:
Done.